### DISCUSSION TITLE
Error installing latest version

### DISCUSSION THREAD

**Discussion by stgblack** [johnwmillr/LyricsGenius#223] [Q&A]
Thank you so much for making this publicly available! I am new to python and I was trying to install the latest version of this library because if it seems that I cannot access the API without it. However, when I attempt to install the latest version, I get the following error

![johnw genius pip install error](https://user-images.githubusercontent.com/82466995/134821085-b9eb60b0-3f77-43ee-8dad-8ad94c2acd75.JPG)
 
Could this be because there has been another version since the link has been updated? Thank you so much for your help.

**Comment by johnwmillr** [comment]
Sorry for the late response. It looks like the error message is saying you don't have `git` installed on your machine. You can try installing `git`, or try this command in a different cell: `!pip install lyricsgenius`.
